draft-ietf-jsonpath-base.md

```diff
@@ -1520,8 +1519,7 @@ The well-typedness of function expressions can now be defined in terms of this t
 A function expression is well-typed if all of the following are true:
 
 * If it occurs directly in a test expression, the function is declared
-  to have a result type of `LogicalType`, or (conversion applies)
-  `NodesType`.
```

Comment:
I think this change was meant for below.  `NodesType` is still valid for test expressions.  The line below is about comparisons.